app/services/payout/gemini_service.rb

```diff
@@ -1,62 +1,52 @@
 module Payout
-  class GeminiService < Service
-    def perform
-      # Writes a message if we should skip a publisher or not.
-      # Implemented in Payout::Service
-      return if skip_publisher?
+  class GeminiService
+    def perform(payout_report:, publisher:)
```

Comment:
These were part of the same `Payout::Service` class to use the same initializer so we didn't have to repeat ourselves, may we revert back?

Reply:
This is to bring these in line with the `Bitflyer Service` class for consistency. The methods in that class are utility classes and can be called as such; there's no need to bind these classes together in a hierarchy -- they can each just call the helper. Having it use inheritance causes a lot of pain when trying to add the `Bitflyer Service` class due to the complexity in that class, so we began refactoring it then. This is the completion of that work.